providers/flagd/pkg/service/service.go

```diff
@@ -1,40 +1,66 @@
 package service
 
 import (
+	schemaConnectV1 "buf.build/gen/go/open-feature/flagd/bufbuild/connect-go/schema/v1/schemav1connect"
```

Comment:
This is a nit, but now that you've isolated most of the gRPC code here, you may consider calling this file grpc_service... not necessary though.

If you do, I would recommend you do it with `git mv ...` so that it's easier to see in review.

Reply:
Good suggestion. But I would like to defer this renaming to a follow-up PR. This one already has quite a lot of changes (IMO too many for a single pr 😞 ) so changing the packaging could confuse you even more.